pipelined/bevy_render2/src/render_component.rs

```diff
@@ -137,19 +161,23 @@ impl<T: Asset> ExtractComponent for Handle<T> {
     }
 }
 
+/// This system extracts all components of the corresponding [`ExtractComponent`] type.
 pub struct ExtractComponentSystem<C: ExtractComponent>(PhantomData<C>);
 
 impl<C: ExtractComponent> RunSystem for ExtractComponentSystem<C>
 where
     <C::Filter as WorldQuery>::Fetch: FilterFetch,
     <C::Query as WorldQuery>::Fetch: ReadOnlyFetch,
 {
+    /// Specifies all required ECS data.
```

Comment:
Re-documenting trait impl functions with copies of the source docs has net-negative value in my opinion:
1. It requires updating every implementation whenever the source trait changes
2. It just takes up space.

We should only add documentation to specific impls when it actually adds new context. 

Can you take a pass and remove cases like this? (and in cases like this where the main Trait docs are missing, move the docs there)